geopandas/base.py

```diff
@@ -894,7 +894,7 @@ def covers(self, other):
         other : Geoseries or geometric object
             The Geoseries (elementwise) or geometric object to check is being covered.
         """
-        return _binary_geo("covers", self, other)
+        return _binary_op("covers", self, other)
```

Comment:
All the other predicates use `_binary_op`, as `_binary_geo` feeds the result to a `GeoSeries(...)` constructor